styx-scheduler-service/src/main/java/com/spotify/styx/StyxScheduler.java

```diff
@@ -396,11 +397,10 @@ public void create(Environment environment) {
     final DockerRunner dockerRunner = MeteredDockerRunnerProxy.instrument(
         TracingProxy.instrument(DockerRunner.class, routingDockerRunner), stats, time);
 
-    final FlyteRunner flyteRunner = flyteRunnerFactory.create(
+    final FlyteRunner flyteRunner = FlyteRunner.routing(
+        id -> flyteRunnerFactory.create(id, environment.config(), stateManager),
         //TODO: define globalFlyteRunnerId in styxConfig
```

Comment:
Should we fix this in this PR or create a new one?

Reply:
nice catch sonja. Let's fix this on the next one.

Reply:
I am working on the follow up PR for this TODO